providers/ldap/ldap_client.go

```diff
@@ -95,7 +95,7 @@ func (l *LClient) newConn() (*ldap.Conn, error) {
 		certFound := false
 		for _, certFile := range certFiles {
 			if _, err = os.Stat(certFile); os.IsNotExist(err) {
-				log.Warning("CA cert file %s is not present", certFile)
+				log.Warningf("CA cert file %s is not present", certFile)
```

Comment:
Error out, do not just warn and move on the ca-certs file should always be present. It is insecure to continue on if it isnt